Capstone/View Controllers/HomeViewController.m

```diff
@@ -0,0 +1,102 @@
+//
+//  HomeViewController.m
+//  Capstone
+//
+//  Created by Abel Kelbessa on 7/4/22.
+//
+
+#import "HomeViewController.h"
+#import "HomeCell.h"
+#import "UIImageView+AFNetworking.h"
+#import "DetailsViewController.h"
+
+@interface HomeViewController ()<UITableViewDelegate, UITableViewDataSource>
+
+@property(strong, nonatomic) NSDictionary *allCollegeInfo;
+@property(strong, nonatomic) NSArray *colleges;
```

Comment:
what is this an array of? I *think* this is an array of dictionaries, but I can't be sure because there's no mentioned type.

including types is not just useful for legibility, but it also prevents type issues. i.e. you can accidentally insert a string in the `colleges` array instead of an NSDictionary

```suggestion
@property(strong, nonatomic) NSArray<NSDictionary *> *colleges;
```